src/core/collections.js

```diff
@@ -12,18 +12,21 @@ export let collections = new Map();
 
 const mergeDefaults = itemDefaults => item => ({ ...item, ...itemDefaults?.find(def => def.id === item.id) });
 const getFilterParams = config => ({ ids: config.defaults?.map(x => x.id) ?? [], tags: config.include ?? [] });
-const filterItems = config => item => config.ids.includes(item.id) || item.tags.some(tag => config.tags.includes(tag));
-const addQty = qty => item => ({ ...item, qty });
+const tagIn = config => tag => config.tags.includes(tag);
+const include = config => item => !config.tags.length || config.ids.includes(item.id) || item.tags?.some(tagIn(config));
+const addQty = qty => item => ({ ...item, qty }); //TODO this should be add global defaults. qty might not even be present? needed for shop
```

Comment:
It feels a little bit odd for states to have a quantity. I know it's not really going to hurt us (after all, there's still one of a unique thing even if you wouldn't usually think of that thing as existing as a quantity) so it might just be a question of tolerating a bit of semantic dissonance in the name of simplicity....

Reply:
Yeah totally agree. I had 2 things to discuss on this:
* Rename "items" folder to collections?
* Apply any defaults - not just qty

I think the defaults should just be anything and then you could miss them out for select screens and so on.
Gonna stick that in another small follow up ticket though as this was mainly about removing the states file.
Prob needs some docs updates too come to think of it...